.github/workflows/build-and-test.yml

```diff
@@ -15,12 +15,13 @@ env:
 jobs:
   clippy_check:
     runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
     env:
       RUSTFLAGS: "-Dwarnings" # Make sure CI fails on all warnings, including Clippy lints
     steps:
       - uses: actions/checkout@v4
       - name: Run Clippy
-        run: cargo clippy --all-targets --all-features
+        run: cargo clippy --all-targets --all-features -vv
```

Comment:
Curious why you added `-vv` here? For debugging purpose?

Reply:
[52f34ac](https://github.com/zkmopro/mopro/pull/318/commits/52f34ac21746bf2fcfc4721882f2985158c82dad)